Tasks/Common/PowershellHelpers/Helpers.ps1

```diff
@@ -27,7 +27,7 @@ function Invoke-ActionWithRetries
         $RetryMessage
     )
 
-    Trace-VstsEnteringInvocation $MyInvocation
+    Write-Host (Get-VstsLocString -Key EnteringInvokeActionWithRetries)
```

Comment:
This will get printed everytime. may be don't keep this.

Reply:
Okay